File: packages/sandbox/src/fixtures/echo.js
const inputPath = "/scratch/input.txt";
let inputText = "";
try {
  inputText = require("node:fs").readFileSync(inputPath, "utf-8");
} catch {
  // No input file provided for this invocation — that's fine, just echo args.
}
console.log(JSON.stringify({ args: process.argv.slice(2), inputText }));

File: packages/sandbox/src/fixtures/echo.py
import json
import sys

input_path = "/scratch/input.txt"
input_text = ""
try:
    with open(input_path, "r", encoding="utf-8") as f:
        input_text = f.read()
except FileNotFoundError:
    pass

print(json.dumps({"args": sys.argv[1:], "inputText": input_text}))
